libs/shared/components/src/Markdown/Markdown.tsx

```diff
@@ -1,11 +1,24 @@
 import { compiler } from 'markdown-to-jsx'
-import { markdownOptions } from './markdownOptions'
+import { getMarkdownOptions } from './markdownOptions'
 
 interface Props {
   children: string
+  conditionals?: OptionalConditionalOverrides
```

Comment:
maybe rename to "options", conditionals doesn't really describe what we are doing here I feel like

Reply:
fair, fixed